### PR TITLE
added best practices for using shared folders in documentation

### PR DESCRIPTION
**Pull request by NivKeidan**
Shared folders usage in Anka is not optimized, so it is best to point that out to users

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Comment by tduffield** [comment]
Thanks for the PR. This is interesting -- we've never had any issue with this. Do you have a link to the specific guidance?

**Comment by NivKeidan** [comment]
As a Veertu employee, I can confirm this is a best practice presented by us.
(This has recently been discussed in our #general slack chan by @NorseGaud as well)

Using shared folders is not necessarily error prone, but it is not optimized.
Build tasks, for example, over shared folders can take more time significantly.

This PR has come due to increasing amount of questions and issues surrounding this.

**Comment by tduffield** [comment]
@NivKeidan gotcha. I'm happy to accept this PR if you're able to DCO sign off your commit.

**Comment by NivKeidan** [comment]
Done

Thank you!!

File: README.md [hunk 1]
```diff
@@ -16,6 +16,24 @@ steps:
...
+## Best Practices
+### Shared Folders
+
+[Veertu](https://veertu.com) states that the performance of using shared folders is not completely optimized, so it is best practice to disable this.
+As an alternative, it is suggested to clone and pull the repository as the first commands in the pipeline step.
+
+Example:
+```yml
+steps:
+  - command: "git clone $BUILDKITE_REPO && cd repo-folder && git checkout -f $BUILDKITE_COMMIT"
```
Comment (tduffield) [inline README.md]: Looking at this closer, I'm not sure this is correct syntax? I think the following is "more correct" (?)

```
steps:
  - commands:
      - git clone $BUILDKITE_REPO && cd repo-folder && git checkout -f $BUILDKITE_COMMIT
      - cd repo-folder; ./build.sh
    plugins:
      - thedyrt/skip-checkout#v0.1.1: ~
      - chef/anka#v0.5.5:
          vm-name: base-vm-mojave
          no-volume: true
          wait-network: true
```

Comment (NivKeidan) [reply README.md]: 💯 
Thanks for putting in the time

File: README.md [hunk 2]
```diff
@@ -16,6 +16,25 @@ steps:
...
+## Best Practices
+### Shared Folders
+
+[Veertu](https://veertu.com) states that the performance of using shared folders is not completely optimized, so it is best practice to disable this.
+As an alternative, it is suggested to clone and pull the repository as the first commands in the pipeline step.
+
+Example:
+```yml
+steps:
+  - command: git clone $BUILDKITE_REPO && cd repo-folder && git checkout -f $BUILDKITE_COMMIT
```
Comment (tduffield) [inline README.md]: We can't have two top-level `command` steps like this. Buildkite will read this as two separate jobs, only one of which will have the clone commands. This needs to be nested under a single `command` as in my initial comment.

Comment (NivKeidan) [reply README.md]: Updated.

Thanks again

**Pull request merged**